{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/views.py

```diff
@@ -3,5 +3,6 @@
 
 @view_config(route_name='home', renderer='templates/mytemplate.jinja2')
 @view_config(route_name='home', renderer='templates/mytemplate.pt')
+@view_config(route_name='home', renderer='/mytemplate.mako')
 def my_view(request):
```

Comment:
This deviates from the established pattern of `templates/mytemplate.<ext>`. If you follow the pattern, then you would not need to modify either `development.ini` or `production.ini`.